db/schema.rb

```diff
@@ -19,8 +20,6 @@
     t.text "text"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.bigint "user_id", null: false
-    t.bigint "post_id", null: false
     t.index ["post_id"], name: "index_comments_on_post_id"
     t.index ["user_id"], name: "index_comments_on_user_id"
```

Comment:
- [ ] From the ERD [diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png?raw=true) provided, it is seen that the comment, post, and the like has a foreign key `author_id` that references the author in the users table but what you have here is a foreign key `user_id` that references the `users` table. Kindly fix this by creating a new migration and correcting foreign keys or by replacing the classes for adding User references with the code below in the migration files: After that please run `rails db:migrate` to generate new data in the schema file having the correct references as provided in the project requirement.

```ruby
  class AddUsersRefToComments < ActiveRecord::Migration[6.1]
  def change
    add_reference :comments, :author, foreign_key: { to_table: :users }
  end
end
```

```ruby
   class AddUsersRefToPosts < ActiveRecord::Migration[6.1]
  def change
    add_reference :posts, :author, foreign_key: { to_table: :users }
  end
end
```

```ruby
  class AddUsersRefToLikes < ActiveRecord::Migration[6.1]
  def change
    add_reference :likes, :author, foreign_key: { to_table: :users }
  end
end
```